Description: Fix FTBFS.
 applet.vala:291.23-291.29: error: `Timeout' is an ambiguous reference between `GLib.Timeout' and `Gtk.Timeout'
      this.timer_id = Timeout.add (300, this.on_timer_tick);
Author: Miguel A. Colón Vélez <debian.micove@gmail.com>
Forwarded: no

--- a/applets/quick-prefs/applet.vala
+++ b/applets/quick-prefs/applet.vala
@@ -288,7 +288,7 @@
 
     if (this.timer_id == 0)
     {
-      this.timer_id = Timeout.add (300, this.on_timer_tick);
+      this.timer_id = GLib.Timeout.add (300, this.on_timer_tick);
     }
     if (this.autohide_cookie == 0)
     {
